src/parseTools.js

```diff
@@ -429,6 +429,14 @@ function makeHEAPView(which, start, end) {
   return `HEAP${which}.subarray((${start})${mod}, (${end})${mod})`;
 }
 
+// Similar to makeHEAPView, useful when you do not want to compute the `end` value.
+function makeHEAPViewWithCount(which, start, count) {
```

Comment:
It looks like almost all callers of `makeHEAPView` what this form anyway.. so maybe we can just convert the existing function..  looking into how feasible that might be..

Reply:
Done.